install.js

```diff
@@ -145,3 +145,17 @@ function generateProtocolTypesIfNecessary(updated) {
     return;
   return require('./utils/protocol-types-generator');
 }
+
+function log(toBeLogged) {
+  function isTrue(value) {
+    return !!value && value !== '0' && value !== 'false';
+  }
+
+  const envDisable = isTrue(process.env.CI);
```

Comment:
Is it standard practice to disable logs when CI is set? I find the logs useful at least on our CI.

Reply:
I agree with you and I’ll remove that part.